src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,37 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
     public static void main(String[] args) {
-        // test your code here
+        Injector injector = Injector.getInstance("mate.jdbc");
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+        Driver driver1 = new Driver("Bob", "123456");
+        Driver driver2 = new Driver("Alice", "654321");
+        Driver driver3 = new Driver("John", "123654");
```

Comment:
It's better not to use numbers in variables names

Reply:
Thought since it's for the testing purposes it would be a "state of nonsense" to care about.
Refactored code anyway, added using method Create()'s returned value.
Thanks!

Reply:
Ideally, our code should be tested using unit tests, and there, we should follow the code style, so let's stick to it in Main class too 🙂